bundles/org.openhab.binding.sunsynk/pom.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>4.2.0-SNAPSHOT</version>
```

Comment:
```suggestion
    <version>4.3.0-SNAPSHOT</version>
```

Reply:
@wborn, I can make the change but it stops compilation, I  guess I  need to move my local  code base on to 4.3.0.
I'm new to all of this how do I  achieve that?

Reply:
Update your fork from openHAB main and then update your feature branch from your fork.

Reply:
Done